src/components/Navbar/index.spec.tsx

```diff
@@ -264,10 +152,40 @@ describe(__filename, () => {
       expect(root.find(VersionChooser)).toHaveLength(0);
     });
 
-    it('does not render a link to the legacy file viewer', () => {
-      const root = render();
+    it('renders a link to the legacy file viewer in the browse page', () => {
```

Comment:
Nothing in this test case makes me think that we are in the browse page. Shouldn't it be something like: `...file viewer when only the currentVersionId is set`?